lib/db/mysql/schema.sql

```diff
@@ -46,7 +46,8 @@ CREATE TABLE IF NOT EXISTS tokens (
   type VARCHAR(16) NOT NULL,
   scope VARCHAR(256) NOT NULL,
   createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  expiresAt TIMESTAMP NOT NULL
+  expiresAt TIMESTAMP NOT NULL,
+  INDEX idx_expiresAt(expiresAt),
```

Comment:
> Update lib/db/mysql/schema.sql to reflect the existence of the index and stored procedure, so that it remains the canonical overview of the desired state of the db

@rfk Not sure what you meant by showing existence of stored procedure? Did you want to have `CREATE PROCEDURE PURGE_EXPIRED_TOKENS` statement here? Or did you want to have the entire stored procedure?